app/lib/common/widgets/acter_icon_picker/acter_icon_widget.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:acter/common/widgets/acter_icon_picker/picker/acter_icon_picker.dart';
+import 'package:acter/common/widgets/acter_icon_picker/model/acter_icons.dart';
+import 'package:flutter/material.dart';
+
+class ActerIconWidget extends StatefulWidget {
```

Comment:
This class is for generic ActerIcon widget which is used in multiple places like list item, edit/selection mode on creation form etc.. 


1) Used in Pin Create Form:
![Screenshot 2024-09-05 at 1 49 47 PM](https://github.com/user-attachments/assets/b46d10ed-d96c-4293-90fa-e80dcf3456fb)

2) Used in PinList Item:
![Screenshot 2024-09-05 at 1 50 53 PM](https://github.com/user-attachments/assets/c1808d47-04e9-4f72-a3a8-94ca47a61be5)

3) Used in Pin Detail screen:
![Screenshot 2024-09-05 at 1 51 27 PM](https://github.com/user-attachments/assets/bcaf652d-9574-49f5-ab7b-3abb9087e21a)

4) Used in TaskList Create Form:
![Screenshot 2024-09-05 at 1 50 06 PM](https://github.com/user-attachments/assets/8aacc624-2c8b-4d6f-b279-724930b0545d)

5) Used in TaskList Item:
![Screenshot 2024-09-05 at 1 52 37 PM](https://github.com/user-attachments/assets/ffd92300-93d4-4791-8590-e044ce5969da)

6) Used in TaskList Detail screen:
![Screenshot 2024-09-05 at 1 52 48 PM](https://github.com/user-attachments/assets/18baa7cc-91a4-486e-849a-cf8c3f251c08)